spec/models/shelter_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe Shelter do
   describe "relationships" do
     it {should have_many :pets}
```

Comment:
Spacing improvements are needed here too, but it can be done at the end as well.